src/results.js

```diff
@@ -18,14 +18,17 @@ export default ({ service, tree, totalPages = 100, ...node }) => {
   let scrollId
 
   let result = {
-    getTotalRecords: async () => {
+    include: node.include,
```

Comment:
It seems like we should create a first class `columnOrder` prop which becomes `headers` in the fast-csv API with write headers false (because we will render them ourselves, respecting `columnOrder`